lib/scenarios/utils.js

```diff
@@ -23,7 +23,7 @@ export async function clearCaches(directory) {
 
 export async function getDirSizes(directory) {
 	let size = 0;
-	await runCommand("du", ["-sb", directory], {
+	await runCommand("du", ["-s", directory], {
```

Comment:
-b does not exists on Mac and linux, I am not sure if it should be delete

Reply:
-b means return bytes size, you can remove it, but do not forget to set the size to size*1024 on line 28

Reply:
I got error when run `node bin/bench.js` on my own mac if this parameter is not removed. So I prefer to add 1024 * size.